src/modules/direnv.rs

```diff
@@ -45,6 +45,7 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 "rc_path" => Some(Ok(state.rc_path.to_string_lossy())),
                 "allowed" => Some(Ok(match state.allowed {
                     AllowStatus::Allowed => Cow::from(config.allowed_msg),
+                    AllowStatus::NotAllowed => Cow::from(config.denied_msg),
```

Comment:
Alternatively, we could skip adding `not_allowed_msg` for now
```suggestion
                    AllowStatus::NotAllowed => Cow::from(config.not_allowed_msg),
```

Reply:
🤦‍♂️ dang, another good catch

I'd be fine with that too - I don't remember the exact distinction of NotAllowed/Denied but I _think_ it's when you have explicitly denied a file vs not having been granted. 

_Seems_ worthwhile for people to decide if they care about the distinction but for my use, I have them configured in a way that it doesn't matter 🤷‍♂️